src/common/metrics/case_density.tsx

```diff
@@ -106,12 +113,13 @@ function renderStatus(projections: Projections): React.ReactElement {
     <Fragment>
       Over the last week, {locationName} has averaged {newCasesPerDayText} new
       confirmed cases per day (<b>{formatDecimal(currentCaseDensity, 1)}</b> for
-      every 100,000 residents). Over the next year, this translates to around{' '}
-      {formatEstimate(newCasesPerYear)} cases and an{' '}
+      every 100,000 residents). If today's trend continued, this would translate
+      over the next year to around {formatEstimate(newCasesPerYear)} cases and
```

Comment:
I feel like this sentence would read better if we moved "over the next year" to the end.
Also "around" feels a bit informal?

Perhaps:

> If today's trend continued, this would translate to approximately [cases] cases and an estimated [infections] ([x]% of the population) infections over the next year.

But if this copy has already been fine-tuned / approved by somebody else (or if you just don't like my proposal), feel free to just ship this.